### ISSUE TITLE
PDF map that is generated is all black when in 3D mode

### ISSUE THREAD

**Issue by BerryDaniel** [GeoNode/geonode-client#171]
When you click on print in layer or map details page, the resulting PDF map that is generated is all black when in 3D mode.

**Comment by bartvde** [comment]
I'm guessing the only way out of this is to disable printing when in 3D mode

**Comment by BerryDaniel** [comment]
That was my thought too

**Comment by bartvde** [comment]
Yeah just checked the code, ol-cesium hides the 2D map: https://github.com/openlayers/ol-cesium/blob/2accc3ff7770cbe3914a71d3bdc122d6f2de88ad/src/olcesium.js#L543:L547

**Comment by bartvde** [comment]
Fixed in SDK with https://github.com/boundlessgeo/sdk/pull/413
